rs_db/src/main/java/com/luxoft/jmswithspring/database/dao/TableCreator.java

```diff
@@ -0,0 +1,41 @@
+package com.luxoft.jmswithspring.database.dao;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import static com.luxoft.jmswithspring.database.dao.TableCreatorConstants.*;
+
+public enum TableCreator {
+
+    USER(DROP_TABLE_USER, CREATE_TABLE_USER),
+    TRANSACTION(DROP_TABLE_TRANSACTION, CREATE_TABLE_TRANSACTION),
+    BRANCH(DROP_TABLE_BRANCH, CREATE_TABLE_BRANCH),
+    TRAN_LOTS(DROP_TABLE_TRAN_LOTS, CREATE_TABLE_TRAN_LOTS),
```

Comment:
I have check structure, I do not understated way do you need in this table. https://smehrozalam.files.wordpress.com/2010/06/erd-many-to-many-1.jpg
I think you need allowed_sec..._for_bran...